bftengine/src/bftengine/ViewsManager.cpp

```diff
@@ -568,13 +569,17 @@ bool ViewsManager::tryToEnterView(ViewNum v,
   debugHighestViewNumberPassedByClient = v;
   debugHighestKnownStable = currentLastStable;
 
-  if (currentLastExecuted < currentLastStable)
+  if (currentLastExecuted < currentLastStable) {
     // we don't have state, let's wait for state synchronization...
+    LOG_INFO(GL, "Waiting for state synchronization before entering view ...");
     return false;
+  }
 
-  if (currentLastStable < lowerBoundStableForPendingView)
+  if (currentLastStable < lowerBoundStableForPendingView) {
     // we don't have the latest stable point, let's wait for more information
+    LOG_INFO(GL, "Waiting for latest stable point before entering view ...");
```

Comment:
Does it make sense to also log the specific view number you are referring to?

Reply:
Yes, definitely, will be easier for reviewing the log. Will add.